### DISCUSSION TITLE
HEADS UP! OSGi-WG needs your help regarding specification of web stuff in OSGi

### DISCUSSION THREAD

**Discussion by laeubi** [ops4j/org.ops4j.pax.web#2029] [General]
OSGi spec is on the way to move to jakarta and noticed that there is actually no one in the OSGI-WG to drive the web stuff and if it makes sense to retire these parts from further specification releases[1].

As I know pax-web (and karaf as well) is quite active in this area I'd like to invite you to participate in the specification stuff e.g. commenting at the referenced issue or join the slack-channel and specification calls (next in on wed nov 23 17:00 GMT+1), feedback and participation would really be welcome!

[1] https://github.com/osgi/osgi/issues/499
[2] https://projects.eclipse.org/projects/technology.osgi/contact

**Comment by grgrzybek** [comment]
@laeubi FYI #1802 #1803 is fixed and I'm going to release Pax Web 10 which supports Jakarta EE 9+ with `jakarta.servlet` packages.

#1803 shows that Pax Web 10 passes almost all OSGi CMPN Whiteboard TCK tests except:
* tests that fail because of problems with Tomcat
* tests that fail because of problems with TCK itself (or at least what I think are TCK problems)
* one failure because I didn't implement hook filtering of ServletContextHelper services

**Comment by mattrpav** [comment]
@laeubi what is the Slack channel for the OSGI WG?
